sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -51,7 +51,7 @@ Netty and include OkHTTP client in your pom.xml.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-http-okhttp</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-beta.1</version>
```

Comment:
Same as above, this needs to be left at 1.0.0

Reply:
fixed